### DISCUSSION TITLE
Server not starting when running same code on another computer.

### DISCUSSION THREAD

**Discussion by Hallzmine** [Minestom/Minestom#632] [Help]
My friend is trying to run the exact same code with the same jdk (openjdk 17.0.2) and IDE (Intellij IDEA) as me with the same run config, and it is giving him an error. The code that is being ran is almost identical to the demo code.

We have already tried clearing the cache and replacing the minestom build number with -SNAPSHOT

Here is the build.gradle
```groovy
plugins {
    id 'java'
}

group 'ml.khall.manhattan'
version '1.0'

repositories {
    mavenCentral()
    maven { url 'https://repo.spongepowered.org/maven' }
    maven { url 'https://jitpack.io' }
}

dependencies {
    implementation 'com.github.Minestom:Minestom:bcc652ede8' //You can update this on jitpack.io/#Minestom/Minestom
}
```

And here is the error

```
Error: Unable to initialize main class ml.khall.manhattan.StartServer
Caused by: java.lang.NoClassDefFoundError: net/minestom/server/instance/ChunkGenerator
```

**Comment by TheMode** [comment]
You need to build a fat jar, see https://wiki.minestom.net/setup/your-first-server#build-the-server-jar

**Reply by Hallzmine** [reply]
I don't want him to be just running the jar, he is also going to be developing, so this is not applicable?

**Reply by JustDoom** [reply]
I would get them to test a compiled jar to see if that at least works

**Reply by Hallzmine** [reply]
A compiled jar works on his system

**Comment by SteinGaming** [comment]
Does he have any other JDK's installed? Your gradle.build file doesn't seem to specifically specify jdk-17.

**Reply by Hallzmine** [reply]
JDK 17 is specified within Intellij, he has a jre but no other jdks

**Comment by emortaldev** [comment]
Maybe the actual zip would be useful (obviously not if private code) could mean we can troubleshoot 

(otherwise I would suggest them deleting and redoing the project)
